src/views/blocks/tickets.php

```diff
@@ -133,7 +133,19 @@
 		$this->template( 'blocks/tickets/item-inactive', [ 'is_sale_past' => $is_sale_past ] );
 	endif;
 
-	$this->template( 'v2/components/loader/loader' );
+	ob_start();
+	/**
+	 * Allows filtering of extra classes used on the tickets-block loader
+	 *
+	 * @since  4.11.0
+	 *
+	 * @param  array $classes The array of classes that will be filtered.
+	 */
+	$loader_classes = apply_filters( 'tribe_tickets_block_loader_classes', [ 'tribe-tickets-loader__tickets-block' ] );
+	include Tribe__Tickets__Templates::get_template_hierarchy( 'components/loader.php' );
+	$html = ob_get_contents();
+	ob_end_clean();
+	echo $html;
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$html'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$html'.

<sub>hash: 0b431934e1ffb22c4ae66c5a9fdc2414</sub>